src/main/views/components/questions/template.njk

```diff
@@ -455,14 +455,14 @@
                               <div class="govuk-form-group yeswho">
                                 {{ CCSInput({
                                       label: {
-                                        text: "[ How many resources need a lower security and vetting requirement?]",
```

Comment:
@dhivya-cognizant have you check tis line...... is this is impacting security and vetting requirement? functionality.